### ISSUE TITLE
Code sample tab order should be well defined 

### ISSUE THREAD

**Issue by razor-x** [seamapi/docs#374]
Code sample order depends on the object keys, but those are not ordered. The order should match the order defined [here](https://github.com/seamapi/docs/blob/0cc73a0df5ecd85f7ead3ae9d39f782013b6d6dd/src/lib/layout-context.ts#L10-L16)
